core/src/test/java/io/questdb/griffin/O3CommitLagTest.java

```diff
@@ -684,7 +684,7 @@ private void testCommitLagEndingAtPartitionBoundaryPlus10(CairoEngine engine, Sq
         compiler.compile(sql, sqlExecutionContext);
 
         // i=184 is the last entry in date 1970-01-06
-        sql = "create table y as (select * from x where i<=150) partition by DAY";
+        sql = "create table y as (select * from x where i<=150) timestamp(ts) partition by DAY";
```

Comment:
This is very risky. When we move to async filter execution, records might come out of order. `timestamp(ts)` does not enforce order, so there is a risk of creating table with wrong order of records and declared timestamp. O3 will crash.